doc/make.py

```diff
@@ -309,5 +309,5 @@ def build_all():
                               ' are %s' % (command, funcd.keys())))
         func()
 else:
-    all()
+    raise SystemExit("Missing target.")
```

Comment:
this should be clarified in this case...something like "You must supply a target after `make.py`"